Events.php

```diff
@@ -81,4 +81,12 @@ final class Events
      * @Event("Lexik\Bundle\JWTAuthenticationBundle\Event\JWTExpiredEvent")
      */
     public const JWT_EXPIRED = 'lexik_jwt_authentication.on_jwt_expired';
+
+    /**
+     * Dispatched before the JWE is computed.
+     * This event allow the JWE header parameters to be changed.
```

Comment:
A small note telling that this event is only dispatched when using the web-token integration would be nice.

Reply:
Note added